### ISSUE TITLE
https://tele5.de

### ISSUE THREAD

**Issue by usergittest123** [yt-dlp/yt-dlp#4636]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.08.08** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

Tele5 does not work:
example:
https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/endars-sohn/

[Tele5] endars-sohn: Downloading webpage
[Tele5] 2880911: Downloading token
[Tele5] 2880911: Downloading JSON metadata
[Tele5] 2880911: Downloading JSON metadata
ERROR: [Tele5] endars-sohn: Missing deviceId in context

E:\01_Büro\50_PC\60_VBA\Visual Studio\MedienVersion2.1\bin\x86\Debug\net5.0-windows>

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
debug] Command-line config: ['-vU', 'https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/endars-sohn/']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.08.08 [3157158] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 2021-09-30-git-3ee4502753-full_build-www.gyan.dev (setts), ffprobe 2021-09-30-git-3ee4502753-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.08.08, Current version: 2022.08.08
yt-dlp is up to date (2022.08.08)
[debug] Using fake IP 53.136.149.212 (DE) as X-Forwarded-For
[debug] [Tele5] Extracting URL: https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/endars-sohn/
[Tele5] endars-sohn: Downloading webpage
[Tele5] 2880911: Downloading token
[Tele5] 2880911: Downloading JSON metadata
[Tele5] 2880911: Downloading JSON metadata
ERROR: [Tele5] endars-sohn: Missing deviceId in context
  File "yt_dlp\extractor\common.py", line 666, in extract
  File "yt_dlp\extractor\tele5.py", line 84, in _real_extract
  File "yt_dlp\extractor\dplay.py", line 102, in _get_disco_api_info
  File "yt_dlp\extractor\dplay.py", line 50, in _process_errors
```

**Comment by Vangelis66** [comment]
> I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for **similar issues** including closed ones. DO NOT post duplicates

... If you did search, you failed miserably at it 😉 ...

> ars-sohn: **Missing deviceId in context**

Duplicate of #3623 (and that is deemed to be a duplicate of #3183) ...
TL;DR: 
https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/endars-sohn/
is behind full-blown DRM 😞 ...
